models/OrderDepositTransaction.go

```diff
@@ -16,8 +16,8 @@ func (OrderDepositTransaction) TableName() string {
 	return "OrderDepositTransactions"
 }
 
-// GetOrderDepositTransactionRef returns a reference of
-func GetOrderDepositTransactionRef(transactionID, orderID uint32, isAsk bool) *OrderDepositTransaction {
+// MakeOrderDepositTransactionRef returns a reference of OrderDepositTransaction
+func MakeOrderDepositTransactionRef(transactionID, orderID uint32, isAsk bool) *OrderDepositTransaction {
```

Comment:
unexport....capitalizing makes functions available outside the package (folder), not files.

Reply:
Didn't know. Fixed it now xD